content/go/concepts/strings/terms/fields/fields.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: 'Fields()'
+Description: 'Splits a string into substrings based on whitespace and returns a slice of the substrings.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Fields()`** function in Go is used to split a string into substrings based on whitespace and return a `slice` of the substrings. It removes any leading or trailing whitespace and treats consecutive whitespace characters as a single separator.
+
+## Syntax
+
+```pseudo
+strings.Fields(str)
+```
+
+Where `str` is the original string.
+
+## Example
+
+The following example demonstrates the use of the `strings.Fields()` function.
+
+```go
+package main
+import (
+ "fmt"
+ "strings"
+)
+
+func main() {
+  str := "Hello World Golang"
+  fields := strings.Fields(str)
+
+  fmt.Println(fields)
+}
+```
+
+The output will be:
+
+```shell
+[Hello World Golang]
+```
+
+## Codebyte Example
```

Comment:
The codebyte is exactly the same as the example above. Can we make it different somehow?

Reply:
Sure.

Reply:
@sumitdethe27 I meant doing something different than using the function and just displaying the result. Maybe loop through the `fields` variable and print it that way?

Reply:
Oh!, Okay i got it.